examples/pylint/E0302_unexpected_special_method_signature.py

```diff
@@ -0,0 +1,11 @@
+class A:
+	def __init__(self):
```

Comment:
👆 for all examples👇

correct me if i'm wrong; i believe we are meant to use short descriptive docstrings (with double quotes) in the classes, methods, and functions.